### PR TITLE
CTPPS pixel DQM histograms update

### PR DESCRIPTION
**Pull request by popovvp**

### CODE REVIEW

File: DQM/CTPPS/plugins/CTPPSPixelDQMSource.cc [hunk 1]
```diff
@@ -328,19 +346,28 @@ edm::EventSetup const &)
 }
 
 //-------------------------------------------------------------------------------
-void CTPPSPixelDQMSource::beginLuminosityBlock(edm::LuminosityBlock const& lumiSeg,
```
Comment (dmitrijus) [inline DQM/CTPPS/plugins/CTPPSPixelDQMSource.cc]: Please remove both beginLuminosityBlock and endLuminosityBlock.
Add remove current_LS, you can get in analyze(), "event.getLuminosityBlock()" I think.

Comment (popovvp) [inline DQM/CTPPS/plugins/CTPPSPixelDQMSource.cc]: done

**Comment by dmitrijus** [comment]
+1

**Comment by davidlange6** [comment]
+1

**Comment by threus** [comment]
@popovvp do you have a back port in 92X of this PR? If not, please create it (based on **CMSSW_9_2_5_patch2**) so DQM can put it in online production, as requested by Valentina.

**Pull request merged**